src/QueryAggregator.js

```diff
@@ -109,8 +109,7 @@ export default class QueryAggregator {
       });
     });
 
-    queryConfig.name =
-      ['$$_aggregated', ...Object.keys(queryConfig.queries)].join('-');
+    queryConfig.name = 'AggregatedReactRouterQueryConfig';
```

Comment:
Isn't we still need an unique query config name for a given set of queries, because `getRelayQueries` [uses](https://github.com/facebook/relay/blob/v0.9.3/src/container/getRelayQueries.js#L49) it as a part of the cache key? Otherwise Relay would probably use the set of queries belonging to the old route after changing the route to a new one (in the case the query params stay the same).